src/views/v2/tickets/item/inactive.php

```diff
@@ -61,6 +61,17 @@
 	? sprintf( __( '%s are no longer available', 'event-tickets' ), tribe_get_ticket_label_plural( 'event-tickets' ) )
 	/* translators: %s: Tickets label */
 	: $provider->get_tickets_unavailable_message( $tickets );
+
+/**
+ * Filters the unavailability message for a ticket collection
+ *
+ * @since TBD
+ *
+ * @param $message string Unavailability message
+ * @param $tickets array Collection of tickets
+ */
+$message = apply_filters( 'tec_tickets_unavailable_message', $message, $tickets );
```

Comment:
If the filter is already within `get_tickets_unavailable_message` then we should not need it here again.

Reply:
Also, this will disregard the $is_sale_past message and always show the `tec_tickets_unavailable_message` filtered value when this filter is used.

Reply:
@rafsuntaskin I don't disagree with you. However, by removing the filter from this file we default to `sprintf( __( '%s are no longer available', 'event-tickets' ),`. Shouldn't that be filterable as well?

Reply:
That is within the template, so if anyone wants to change that they can simply overrride the template. Having a filter makes more sense as well, but I just think that's outside the scope for this ticket.

Reply:
@redscar If anything, it should be a new filter. Something other than `tec_tickets_unavailable_message`. Could cause problems filtering it twice.

Reply:
Plus, like @rafsuntaskin mentioned, this is within the template file. It's pretty uncommon to use filters here.